benchmark/alpa/benchmark_3d_infer_one_case_gpt_bert.py

```diff
@@ -199,12 +213,26 @@ def benchmark_gpt_bert_internal(model_type,
     executable.sync()
 
     # Benchmark latency
+    losses = []
     tic = time.time()
+    times = {"start": tic}
     for i in range(niter):
         print(f"Iteration {i} ...")
-        _ = infer_step(params, batch, rngkey)
+        loss = infer_step(params, batch, rngkey)
+        loss.get_remote_buffers_async()  # unstable API
+        losses.append(loss)
+        if not stream_mode:
+            executable.sync()
+    for i, loss in enumerate(losses):
+        print(loss._value)
+        times[f"iter{i}"] = time.time() - tic
+    if stream_mode:
```

Comment:
cc @zhisbug 

With this PR, you can implement ORCA's iteration-level schedule and pipeline parallel schedule following this code snippet.